training/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,29 @@
+Cloud Services and Infrastructure in Digital Publishing
+===========================
+
+We use Amazon Web Services in Digital Publishing to host all of our services, as well as our Continuous Integration (CI).
+
+You should be given relevant AWS accounts by your manager or a Tech Lead - speak to your manager if this does not happen soon upon starting with the team. 
+
+
+Introduction to our Infrastructure
+----------------------------
+
+In your first few weeks you should be invited to a number of 'Intro Workshops', one of which will be on the topic of Infrastructure. 
+
+### Pre reading
```

Comment:
I wonder if it might be worth having both a pre-reps section and a pre-reading one as then we can highlight where one piece of training would benefit from following another and build a sort of series. We can always just put "None" for pre-read for the the intro ones.